src/test/scala/sigmastate/CostingSpecification.scala

```diff
@@ -197,8 +197,10 @@ class CostingSpecification extends SigmaTestingData {
     cost(s"{ $rootTree.isInsertAllowed }") shouldBe (AccessRootHash + selectField)
     cost(s"{ $rootTree.isUpdateAllowed }") shouldBe (AccessRootHash + selectField)
     cost(s"{ $rootTree.isRemoveAllowed }") shouldBe (AccessRootHash + selectField)
-    cost(s"{ $rootTree.updateDigest($rootTree.digest) == $rootTree }") shouldBe (AccessRootHash + selectField + newAvlTreeCost + comparisonPerKbCost)
-    cost(s"{ $rootTree.updateOperations(1.toByte) == $rootTree }") shouldBe (AccessRootHash + newAvlTreeCost + comparisonPerKbCost + constCost)
+    cost(s"{ $rootTree.updateDigest($rootTree.digest) == $rootTree }") shouldBe
+      (AccessRootHash + selectField + newAvlTreeCost + comparisonCost /* for isConstantSize AvlTree type */)
+    cost(s"{ $rootTree.updateOperations(1.toByte) == $rootTree }") shouldBe
+      (AccessRootHash + newAvlTreeCost + comparisonCost + constCost)
```

Comment:
Let's add tests for changed costing rules, they should be quite easy now.

Reply:
only this tests are affected and changed. Others are not affected.
More test will be added in #516